app/services/advance_order_service.rb

```diff
@@ -0,0 +1,35 @@
+class AdvanceOrderService
```

Comment:
we went for option 4 here
https://community.openfoodnetwork.org/t/how-to-name-a-new-service-in-ofn/1530